compiler/src/dmd/location.d

```diff
@@ -28,6 +28,7 @@ enum MessageStyle : ubyte
 {
     digitalmars,  /// filename.d(line): message
     gnu,          /// filename.d:line: message, see https://www.gnu.org/prep/standards/html_node/Errors.html
+    sarif         /// JSON SARIF output
```

Comment:
It would be nice to link to the documentation for SARIF here like the GNU standards link above

Reply:
Sure, will do!

Reply:
done- https://github.com/dlang/dmd/pull/16986/commits/db26e1813ad79be34514c1be562c8f12afcbc1e6